.github/workflows/stale.yaml

```diff
@@ -0,0 +1,24 @@
+name: Stale Pull Requests
+on:
+  # schedule:
+  #   - cron: '0 09 * * *'  
+  workflow_dispatch:
```

Comment:
@NaveenKundeti-IBS  : This key does not need any value?

Reply:
Sir, It needs a `cron` value to run the "stale" actions. Please suggest a time to run these actions. I'll uncomment the `schedule` event and update the workflow @tomegathericon

Reply:
Yes @tomegathericon sir, the recommended time to run the scheduled actions.

Reply:
I am asking for recommendations from you @NaveenKundeti-IBS

Reply:
I would recommend scheduling to run every day at 9:00 AM IST @tomegathericon sir

Reply:
@tomegathericon Sir, this is updated now!